.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -1,5 +1,6 @@
 ---
 name: Bug Report
+labels: bug
```

Comment:
If we could move this line below the `about` line, that'd be great. Not a huge deal, though.